content/php/concepts/math-functions/terms/hypot/hypot.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'hypot()'
+Description: 'Returns the length of the hypotenuse of a right-angle triangle.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Geometry'
+  - 'Math'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`hypot()`** function returns the length of the hypotenuse of a right-angle triangle. Given a right-angle triangle with sides `$length1` and `$length2`, it returns the square root of (`$length1`<sup>2</sup> + `$length2`<sup>2</sup>).
+
+The function accepts two `float` inputs and returns a value of type `float`.
```

Comment:
I would recommend moving this after line 27

Reply:
@caupolicandiaz I have implemented this change in the most recent commit.